torch/_inductor/config.py

```diff
@@ -67,6 +67,9 @@
 # How to import torchdynamo, either torchdynamo or torch.dynamo
 dynamo_import = inductor_import.replace("inductor", "dynamo")
 
+# Pad input tensors of matmul/bmm/addmm to leverage Tensor Cores in NVIDIA GPUs
+shape_padding = os.environ.get("TORCHINDUCTOR_SHAPE_PADDING", "0") == "1"
```

Comment:
what's stopping us from turning this on by default ?

Reply:
Hi @eellison, we tested shape padding optimization in a few internal and OSS models, and we will make it by default when the optimization is
- tested in most internal and OSS benchmarking models;
- supported other dtype (e.g., FP16 for inference) with different heuristics